bin/parse-conf.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
```

Comment:
Executes the templating typically done by the nginx buildpack, substitutes environment variables for:
1. `{{port}}`
2. `{{env "FOOBAR"}}`